tosan-httpserver-spring-boot-starter/src/main/java/com/tosan/http/server/starter/configuration/HttpServerUtilConfiguration.java

```diff
@@ -124,4 +126,16 @@ public ServiceLoggingConfig serviceLoggingConfig() {
         serviceLoggingConfig.setIgnoredParameterTypes(ignoredParameterTypes);
         return serviceLoggingConfig;
     }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public StatisticsAspect statisticsAspect(StatisticsUtil statisticsUtil) {
```

Comment:
no config for aspect for now. so @ConditionalOnMissingBean does not seem necessary